commit 35271ee3f4fc8b90dd9fa538605c72459e8d1909 (refs/changes/21/521/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-22T15:26:18-07:00 (3 years, 1 month ago)
    
    CAPI-522 update node-ufds to support node v4
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/package.json b/package.json
index b586297..9fa1a88 100644
--- a/package.json
+++ b/package.json
@@ -12,9 +12,9 @@
   "dependencies": {
     "assert-plus": "^0.2.0",
     "async": "~0.9.0",
-    "bunyan": "^1.3.3",
+    "bunyan": "^1.8.1",
     "clone": "~0.2.0",
-    "ldapjs": "git://github.com/mcavage/node-ldapjs.git#aed6d2b043715e1a37c45a6293935c25c023ebce",
+    "ldapjs": "0.8.0",
     "libuuid": "0.2.1",
     "lru-cache": "^2.5.0",
     "once": "^1.3.1",
